spec/unit/installer/xcode/target_validator_spec.rb

```diff
@@ -435,6 +435,24 @@ def create_validator(sandbox, podfile, lockfile)
              'any of targets (`TestRunner`) integrating it.'
           end
 
+          it 'does not crash if targets are missing' do
+            fixture_path = ROOT + 'spec/fixtures'
+            config.repos_dir = fixture_path + 'spec-repos'
+            podfile = Podfile.new do
+              project(fixture_path + 'SampleProject/SampleProject').to_s
+              platform :ios, '10.0'
+              install! 'cocoapods', :integrate_targets => false
+              pod 'MultiSwift', :path => (fixture_path + 'multi-swift').to_s
+              supports_swift_versions '< 3.0'
+              target 'SampleProject'
```

Comment:
This seems to have a target?

Reply:
Fixed, Unlike the real code, the create_validator method didn't create pod targets for the validator when there were no targets in the Podfile.